src/lib/chat/matrix/chat-message.ts

```diff
@@ -35,9 +36,11 @@ export async function mapMatrixMessage(matrixMessage, sdkMatrixClient: SDKMatrix
   const parent = matrixMessage.content['m.relates_to'];
   const senderData = sdkMatrixClient.getUser(senderId);
 
+  const parsedContentBody = parsePlainBody(content.body);
```

Comment:
@dalefukami - does this look right to you? While debugging the failing test I noticed that if I log `content.body` it sometimes logs the SecureBackup file code!

<img width="753" alt="Screenshot 2024-03-25 at 17 36 33" src="https://github.com/zer0-os/zOS/assets/39112648/da589d76-04d7-46de-8d0f-172fe27ea012">

Reply:
Fixed this, but, still very strange that the secure backup file gets logged if you console log `content.body` in `mapMatrixMessage`